packages/models/cardstack/code-generator.js

```diff
@@ -54,6 +54,13 @@ define('@cardstack/models/generated/{{modelName}}', ['exports', '@cardstack/mode
         }),
        {{/if}}
      {{/each}}
+
+     resourceMetadata: Ember.inject.service(),
```

Comment:
@ef4 Not sure how we can import `inject` and `computed` so that we can use the direct imports and potentially benefit from code splitting down the line. (And I'm also uncertain whether that matters at this point.)